pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -26,36 +26,31 @@ social:
 %div.starwars-container
   %div.img-container
     %img.starwars-img{src:"/images/sw_landingpage.jpg"}
-  %h1.starwars-heading Building a Galaxy with Code
+  %h1.starwars-heading= I18n.t(:starwars_subtitle)
 
   .col-25.starwars-tutorial{style: "right: 20px;"}
     %div.tutorial-box
-      %h2.sw-h2 JavaScript
+      %h2.sw-h2= I18n.t(:starwars_javascript_title)
       %div.sw-tutorial-info
-        %p.sw-tutorial-description
-          Use drag-drop blocks and JavaScript.
-        %p.sw-tutorial-specs
-          English only | Modern browsers | Ages 11+
+        %p.sw-tutorial-description= I18n.t(:starwars_javascript_description)
+        %p.sw-tutorial-specs= I18n.t(:starwars_javascript_specs)
```

Comment:
its a slight pity to not be using the same mechanism to generate these strings as /learn does, but i think this makes sense, especially as we tweak this page, and keep its behaviour independent of /learn (for now)